tests/vmi_multus_test.go

```diff
@@ -131,27 +162,15 @@ var _ = Describe("Multus", func() {
 
 		configureNodeNetwork(virtClient)
 
-		result := virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "linux-bridge-net-vlan100")).
-			Body([]byte(fmt.Sprintf(linuxBridgeConfCRD, "linux-bridge-net-vlan100", tests.NamespaceTestDefault))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
+		err := createNetworkAttachmentDefinition("linux-bridge-net-vlan100", tests.NamespaceTestDefault, linuxBridgeNADConfig)
```

Comment:
maybe use const for the name. then you can use it also in the error message one row below